feat(flags): Consolidate feature flags pull requests (#12005)
getsentry/sentry-docs: docs/product/explore/feature-flags/index.mdx

Reviewer: taking note that this is inside Explore

Reviewer: This page looks done to me! :shipit:

---
title: "Feature Flags"
sidebar_order: 100
description: "Learn how to set up and interact with Sentry's feature flag evaluation tracking and feature flag change tracking."
---

<PlatformContent includePath="feature-flags/prerelease-alert" />

Enabling a feature flag integration provides deep insights into the state of your application prior to an error. A list of flags and their evaluation results are displayed on each error event in the order of evaluation. Integrating Sentry with your feature flag provider enables Sentry to correlate feature flag changes with new error events and mark certain changes as suspicious.

## Evaluation Tracking

Flag evaluations will appear in the "Feature Flag" section of Issue Details page as a table, with "suspect" flag predictions highlighted in yellow. Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details page documentation](/product/issues/issue-details/#feature-flags).

### Set Up Evaluation Tracking

To set up evaluation tracking visit your provider's integration documentation page:
* [JavaScript](/platforms/javascript/feature-flags/)
* [Python](/platforms/python/feature-flags/)

> ## Change Tracking

Reviewer 2: ```suggestion
## Tracking Changes
```

> 
> Change tracking enables Sentry to listen for additions, removals, and modifications to your feature flags. On change, we'll record the change event in the audit log. The audit log appears in the "event volume" chart and presents itself as a "release" line. If the change is responsible for a new error event, we'll notify you by marking the feature flag as "suspect" on the Issue Details page. Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details documentation](/product/issues/issue-details/#feature-flags).
> 

Reviewer 2: ```suggestion

Change tracking enables Sentry to listen for additions, removals, and modifications to your feature flags. On change, we'll record the change event in the audit log. The audit log appears in the "event volume" chart and presents itself as a "release" line. If the change is responsible for a new error event, we'll notify you by marking the feature flag as "suspect" on the Issue Details page. 

Sentry tracks changes such as additions, removals, and modifications to your feature flags, logging them in the audit log, which appears as a "release" line in the event volume chart. If a change causes a new error, Sentry will mark the feature flag as "suspect" on the **Issue Details** page.

Learn more about feature flag insights by reading about the [Issue Details page](/product/issues/issue-details/#feature-flags).

```

> ### Set Up Change Tracking

Reviewer 2: ```suggestion
### Set Up Change-Tracking
```
